tests/infra/node.py

```diff
@@ -255,7 +256,24 @@ def get_ledger(self):
         return self.remote.get_ledger()
 
     def get_snapshots(self):
-        return self.remote.get_snapshots()
+        # Wait for all available snapshot files to be committed before
+        # copying snapshot directory
+        def wait_for_snapshots_to_be_committed(src_dir, list_src_dir_func, timeout=3):
```

Comment:
Because snapshot evidences are committed asynchronously, it is better to wait for all available snapshots to be committed before attempting to resume from one. This function is a little awkward as it has to work for both `Local` and `SSH` remotes which need to pass the `list_src_dir_func` lambda in.